Tests/CMakeLists.txt

```diff
@@ -225,3 +225,11 @@ if(NOT PELE_ENABLE_EB)
     add_test_r(hit-les-${PELE_DIM}d HITDecay)
   endif()
 endif()
+
+if (PELE_ENABLE_EB)
+  if(PELE_DIM EQUAL 2)
+    if(PELELM_NUM_ODE GREATER 0)
```

Comment:
It looks like this test isn't actually running. I don't think we need the `if(PELELM_NUM_ODE GREATER 0)` - this gets accounted for in the CMakeLIsts.txt for the EB_ODEQty case

Reply:
You're absolutely right.  I've addressed this locally and will push a commit soon.